bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHandler.java

```diff
@@ -178,8 +176,10 @@ protected void updateWemoState() {
                 logger.trace("New state '{}' for device '{}' received", value, getThing().getUID());
                 this.onValueReceived(variable, value, actionService + "1");
             }
+            updateStatus(ThingStatus.ONLINE);
```

Comment:
In case this could be run frequently, you should update status only if not yet ONLINE. 
I let you check that point.

Reply:
Once per minute. What is best practice? I see a lot of bindings update to **ONLINE** unconditionally. It might be better to leave this for being optimized in core (where currently it will always build a new status using builder pattern, so not exactly fast)?

Reply:
If once per minute, I assume it is ok and it will not flood the events log file too much.